README.md

```diff
@@ -168,6 +168,10 @@ use TCP in your application, you need to build the TCP module as mentioned
 above and then create the communication object using CommFactory
 and passing PlainTcpConfig object to it.
 
+We also support TCP over TLS communication. To enable it, change the
+`BUILD_COMM_TCP_TLS` flag to `TRUE` in the main CMakeLists.txt file. When
```

Comment:
Tiny last minute request: add a note in the Dependencies section about installing openssl if using TLS, please.